graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx

```diff
@@ -104,6 +139,15 @@ export default class EnterpriseFreeLicenseForm extends React.Component {
                    placeholder="Please provide a valid email address to send the license key to"
                    required
                    onChange={this.handleInput('email')} />
+            <Input type="text"
+                   id="phone"
+                   label="Phone Number"
+                   value={phone}
+                   placeholder="Please provide a valid phone number"
```

Comment:
You ask for a "valid" phone number, and you parse its validness via a custom regex, but we do not explain to the user what a "valid" phone number looks like, nor do we automatically mask the number to render "validly".